mne_sandbox/preprocessing/_sns.py

```diff
@@ -64,6 +73,11 @@ def fit(self, raw, verbose=None):
         -------
         sns : Instance of SensorNoiseSuppression
             The modified instance.
+
+        Notes
+        -----
+        In the resulting operator, bad channels will be reconstructed by
+        using the good channels.
```

Comment:
okay so if `raw.info['bads']` is empty list, then it's the same as the NoiseTools implementation, and otherwise not?